data/maps/map_files/airplane_2.txt

```diff
@@ -1,24 +1,24 @@
-P 349.81, -501.7633, 125.11, 0, 127, 0, 2, 6-Evians
-P -1108, -516, -15.2926, 0, 0, 127, 2, 2-Swords
-P -306.1557, 749.3103, 767.6651, 99, 0, 255, 2, 7-Swarm
-P 491.8852, 479.3287, 1005.2204, 0, 0, 0, 2, DA_Jump_to_island_3
-P -839, -1419, 0, 0, 0, 127, 2, 2-Swords
+P 349.81, -501.7633, 125.11, 0, 127, 0, 2, 5-Evians
+P -1108, -516, -15.2926, 0, 0, 127, 2, 1.5-Swords
```

Comment:
I don't know if this change will be contentious -- obviously it's been this way for a long time, but the universally accepted numbering on P99 Green is 1, 1.5, 2-8. I don't know if it's the same on Blue, did we just invent a new numbering scheme randomly?